lib/internal/modules/esm/resolve.js

```diff
@@ -615,11 +713,20 @@ function packageImportsResolve(name, base, conditions) {
   throwImportNotDefined(name, packageJSONUrl, base);
 }
 
+/**
+ * @param {URL} url
+ * @returns {PackageType}
+ */
 function getPackageType(url) {
   const packageConfig = getPackageScopeConfig(url);
   return packageConfig.type;
 }
 
+/**
+ * @param {string} specifier
+ * @param {string | URL | undefined} base
+ * @returns {{packageName: string, packageSubpath: string, isScoped: boolean}}
```

Comment:
```suggestion
 * @returns {{ packageName: string, packageSubpath: string, isScoped: boolean }}
```
(exactly 80 chars 😄)